classes/rootfs-image-generator.bbclass

```diff
@@ -72,14 +72,24 @@ create_incremental_update() {
         bbfatal "incremental roofs update is empty"
     fi
 
+    if ${@bb.utils.contains('DISTRO_FEATURES', 'selinux', 'true', 'false', d)}; then
+        set_selinux_context "${ROOTFS_DIFF_DIR}" "${AOS_ROOTFS_SOURCE_DIR}"
+    fi
+
     mksquashfs ${ROOTFS_DIFF_DIR} ${AOS_ROOTFS_IMAGE_FILE} \
         -noappend -wildcards -all-root \
         ${@'' if not d.getVar('AOS_ROOTFS_EXCLUDE_ITEMS') else '-e '+ d.getVar('AOS_ROOTFS_EXCLUDE_ITEMS')}
 }
 
 set_selinux_context() {
-    ROOTFS_FULL_PATH=$(realpath ${AOS_ROOTFS_SOURCE_DIR})
-    setfiles -m -r ${ROOTFS_FULL_PATH} ${ROOTFS_FULL_PATH}/etc/selinux/aos/contexts/files/file_contexts ${ROOTFS_FULL_PATH}
+    target_dir="$1"
+    rootfs_dir="$2"
+
+    TARGET_DIR_FULL_PATH=$(realpath "$target_dir")
```

Comment:
Looks like these vars are local. Use small letters to name them.

Reply:
Done